session-storage/check.js

```diff
@@ -9,3 +21,23 @@ window.setInterval(() => {
   }
 }, 200);
 
+function init() {
+  setRefs();
+  bindEvents();
+}
+function setRefs() {
+  form = document.querySelector('form');
+}
+function bindEvents() {
+  form.addEventListener('submit', submitForm);
+}
+function submitForm(event) {
+  event.preventDefault();
+  const key = document.querySelector('#key').value;
+  const value = document.querySelector('#value').value;
+  if ('world' === key && value) {
+    sessionStorage.setItem('world', value);
+  } else {
+    alert('Either value of key is not "world" or the value is missing.');
```

Comment:
This only allows values to be inserted with the key - 'world'. Is that what the question specified? If so, can you please help me understand the specifics of the question better? Thank you.